content/fact/rcctl.md

```diff
@@ -0,0 +1,17 @@
+---
+title: rcctl(8)
+---
+
+rcctl: the rc.conf.local management tool!
+
+rcctl is a simple utility to maintain rc.conf.local(8), the system daemon
+ configuration database for OpenBSD.
+
+rcctl first appeared in OpenBSD 5.7, in 08/2014.
```

Comment:
Why is rcctl cool? What does it do right? Why do you prefer it instead of others (systemd, sysinitv)?

Reply:
Because, so simply.
- simple to understand
- simple to read
- simple to write
as, KiSS!